src/Sarif/CodeGenHints.json

```diff
@@ -191,6 +191,39 @@
             }
         }
     ],
+    "externalProperties": [
+        {
+            "kind": "BaseTypeHint",
+            "arguments": {
+                "baseTypeNames": [
+                    "PropertyBagHolder"
+                ]
+            }
+        }
+    ],
+    "ExternalProperties.Version": [
+        {
+            "kind": "EnumHint",
+            "arguments": {
+                "typeName": "SarifVersion",
+                "description": "Possible values for the SARIF schema version.",
```

Comment:
>SARIF schema [](start = 56, length = 12)

'for the SARIF external properties schema version'

Reply:
Mike, we have a problem here. there is a single file auto-generated for this enum - SarifVersion.cs. If i modify anything in this node, it ends up overwriting the auto gen file and that in turn breaks the build (because v2tov1 converter needs the OneZeroZero value). should i create a new enum "SarifExternalFileVersion" for this?

---
In reply to: [266152334](https://github.com/Microsoft/sarif-sdk/pull/1342#discussion_r266152334) [](ancestors = 266152334)